docs/_docs/starting-nodes.adoc

```diff
@@ -213,6 +213,28 @@ You can also pass JVM options by using the `-J` prefix:
 ----
 ./ignite.sh -J-Xmx6G -J-DIGNITE_TO_STRING_INCLUDE_SENSITIVE=false
 ----
+
+== Ignite Internal Settings
```

Comment:
Could you please do the following?

- Rename this section to "== Setting Ignite System Properties" and move to the following page (https://ignite.apache.org/docs/latest/setup). Put it after the "Enabling Modules" section.
- Please also move the entire "== Setting JVM Options" section to the same setup page mentioned above.

Reply:
Done.